url/urlsearchparams-constructor.html

```diff
@@ -124,6 +134,30 @@
     params = new URLSearchParams('a%f0%9f%92%a9b=c');
     assert_equals(params.get('a\uD83D\uDCA9b'), 'c');
 }, 'Parse %f0%9f%92%a9');  // Unicode Character 'PILE OF POO' (U+1F4A9)
+
+;[
+  { "input": {"+": "%C2"}, "output": [[" ", "\uFFFD"]], "name": "object with +" },
```

Comment:
That's a good question, it doesn't and I'm not sure it should. I filed https://github.com/whatwg/url/issues/220.